caim_base/forms.py

```diff
@@ -1,24 +1,55 @@
 from django import forms
 from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+from django.db.models import Q
+
 from .models.awg import User
+from .models.geo import ZipCode
+
+
+def zip_validator(zip_code: str):
+    if not ZipCode.objects.filter(Q(zip_code=zip_code)).exists():
```

Comment:
This seems to me to be replicating what a Foreign Key to a ZipCode table would handle, right?

Is this simpler/better than the alternative for some reason I'm unaware of? Or are they pretty much equivalent for this situation.

Reply:
I was following the existing codebase, the Awg model uses a Charfield with a similar validator. 

I think looking at moving both to a foreign key is worth doing at a later date for sure, but in this case I think consistency is key.